### PR TITLE
chore: upgrade flutter to 3.22.1

### PR DESCRIPTION
**Pull request by selfisekai**

### CODE REVIEW

**Comment by krille-chan** [comment]
Thank you so much 💚 but looks like the analyzer already finds the problem I ran into yesterday. The flutter_gen files cannot be found anymore. I have no idea yet how to fix this 🤔

**Comment by selfisekai** [comment]
funny story, i do, it happened to me on f_wallet as well

**Comment by selfisekai** [comment]
wait what

**Comment by selfisekai** [comment]
alright, shit's just on fire https://github.com/flutter/flutter/issues/148333

File: .github/workflows/integrate.yaml [hunk 1]
```diff
@@ -15,6 +15,7 @@ jobs:
           flutter-version: ${{ env.FLUTTER_VERSION }}
           cache: true
       - run: flutter pub get
+      - run: flutter gen-l10n
```
Comment (krille-chan) [inline .github/workflows/integrate.yaml:18]: @selfisekai can you just for fun change this to `dart run build_runner build --delete-conflicting-outputs`? Someone even recommended the funny workaround:

```sh
dart run build_runner build --delete-conflicting-outputs
flutter pub get
dart run build_runner build --delete-conflicting-outputs
```

Comment (krille-chan) [reply .github/workflows/integrate.yaml:18]: nevermind doesnt work

**Comment by selfisekai** [comment]
```
   info • Missing a required trailing comma • lib/pages/chat/events/message.dart:239:59 • require_trailing_commas
   info • Missing a required trailing comma • lib/pages/chat/events/message.dart:319:61 • require_trailing_commas
```
can't reproduce this locally

File: .github/workflows/integrate.yaml (hunk 1 above)
Comment (krille-chan) [inline .github/workflows/integrate.yaml:18]: Do we still need this?

**Pull request merged**